src/components/common/MemosTable/AddReply/showReply.js

```diff
@@ -5,16 +5,14 @@ import moment from 'moment';
 
 function ShowReply(props) {
   const { note_id } = props;
-  const { axiosWithAuth } = useAxiosWithAuth0();
+  const axiosWithAuth = useAxiosWithAuth0();
```

Comment:
in the future, when you come across a linter warning go ahead and update in the file you're working on.